README.md

```diff
@@ -9,8 +9,8 @@
 Sometimes your app needs to fetch information from a third-party API that is slow or costs money. Caching can help! This is a Convex component that can cache the results of expensive functions and set an optional TTL. Expired entries are cleaned up via a cron job once a day. The cache key is the `ActionCache`'s name (defaults to function name) and the arguments to the action that generates the cache values.
 
 ```ts
-import { Client } from "@convex-dev/cache";
```

Comment:
`Client` import is not referenced anywhere
`@convex-dev/cache` package does not exist